poutyne/framework/model.py

```diff
@@ -21,12 +21,12 @@
 
 class Model:
     """
-    The Model class encapsulates a PyTorch module/network, a PyTorch optimizer, a loss function and
+    The Model class encapsulates a PyTorch net/network, a PyTorch optimizer, a loss function and
```

Comment:
```suggestion
    The Model class encapsulates a PyTorch network, a PyTorch optimizer, a loss function and
```

net is only a shortcut for network here.